new/LIT6214MGSocrates.xml

```diff
@@ -0,0 +1,104 @@
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p/>
+            </abstract>
+            <textClass>
+                <keywords>
+                    <term key="ChristianLiterature"/>
+                    <term key="Hagiography"/>
+                    <term key="Miracle"/>
+                </keywords>
+            </textClass>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="SH" when="2020-10-30">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="bibliography">
+
+                <listBibl type="editions">
+                    <bibl>
+                        <ptr target="bm:Budge1930George"/>
+                        <citedRange unit="page">133–135</citedRange>
+                    </bibl>
+                </listBibl>
+
+                <listBibl type="secondary">
+                    <bibl>
+                        <ptr target="bm:Strelcyn1974CatJRylands"/>
+                        <citedRange unit="page">32</citedRange>
+                        <citedRange unit="item">Ms 12, II.7 (6th miracle)</citedRange>
+                    </bibl>
+                    <bibl>
+                        <ptr target="bm:GrebTiss1935Codices"/>
+                        <citedRange unit="page">716</citedRange>
+                        <citedRange unit="item">Vat 233, 3.7 (7th miracle)</citedRange>
+                    </bibl>
+                </listBibl>
+
+                <listRelation>
+                    <relation name="lawd:hasAttestation" active="LIT6214MGSocrates"
+                        passive="BDLaethb4"/>
+                    <relation name="saws:formsPartOf" active="LIT6214MGSocrates"
+                        passive="LIT4006Taamme"/>
+                </listRelation>
+            </div>
+            <div type="edition">
+                <div type="textpart" subtype="incipit" xml:lang="gez">
+                    <note>This incipit is taken from <ref type="mss" corresp="BDLaethb4"/>.</note>
+                    <ab>፮፡ ተአምሪሁ፡ <gap reason="ellipsis" resp="SH"/> ወሀሎ፡ ፩ ብእሲ፡ ክርስቲያናዊ፡ ዘስሙ፡
+                        ስቅራጥስ፡ በኢየሩሳሌም። ወይደዊ፡ ዘስሙ፡ ስላቅ፡ ወቦቱ፡ ወልድ፡ ዘጋኔን።</ab>
+                    <div type="textpart" subtype="explicit" xml:lang="gez">
+                        <note>This explicit is taken from <ref type="mss" corresp="BDLaethb4"
+                            />.</note>
+                        <ab>ወይገብር፡ ለለዓመት፡ ዐቢየ፡ በዓለ፡ ለነዳያን፡ ወለምስኪናን፡ ወለእጓለማውታ፡ ወለፅኑሳን። ወወልዱ፡ ይቀውም፡
```

Comment:
I think this expression should be split into two words (`ወለእጓለ<supplied reason="omitted" resp="SH">፡ </supplied>ማውታ፡`), even though it is sometimes found as one single world

Reply:
Okay, I changed it as suggested because this is the work and not the ms record (I will keep the reading in the ms record)